packages/core/types/src/file/provider.ts

```diff
@@ -144,4 +145,14 @@ export interface IFileProvider {
   getPresignedUploadUrl?(
     fileData: ProviderGetPresignedUploadUrlDTO
   ): Promise<ProviderFileResultDTO>
+
+  /**
+   * Get the file contents as a readable stream.
+   */
+  getAsStream(fileData: ProviderGetFileDTO): Promise<Readable>
```

Comment:
very-nit: Can we call this `getDownloadStream`? 

Two reasons here:
- Aligns the implementation with what we had in v1, so it'll be slightly more intuitive to those coming from the previous major
- Makes the action explicit from the call signature

Reply:
Actually, it's just a stream that you might never use to trigger a download. Like read the stream and process each chunk to perform an import (no download happens)

Reply:
But wouldn't this always download the stream from a remote location, such as with S3?

Reply:
Yup. Makes sense!